hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/ResidentialEnvironmentTransformations.scala

```diff
@@ -28,12 +28,12 @@ object ResidentialEnvironmentTransformations {
     * de_country_[n] is the deprecated user entry free text field
     * de_country_[n]_dd is the updated drop down option list
     */
-  def parseCountry(
+  def parseOnlyCountry(
     rawRecord: RawRecord,
-    countryField: String,
-    countryFieldDropdown: String
+    multipleAdd: String,
+    only1Add: String
   ): Option[String] = {
-    rawRecord.getOptional(countryField).orElse(rawRecord.getOptional(countryFieldDropdown))
+    rawRecord.getOptional(multipleAdd).orElse(rawRecord.getOptional(only1Add))
```

Comment:
it's a little unclear to me how the function name and argument names relate. specifically, a function that parses both fields but is named `parseOnlyCountry` is confusing to me.

could we either change these names or add comments clarifying what each argument of the function is?

Reply:
Added comments to the function to explain the new args